### DISCUSSION TITLE
Install guide for idiot?

### DISCUSSION THREAD

**Discussion by PrincessPotion** [espeak-ng/espeak-ng#1953] [Q&A]
Hi, I'm an idiot. I downloaded the appropriate install file from the current release for my computer and installed; however, running "espeak --version" in powershell states it's not available. What is my dumb brain doing wrong or missing?

**Comment by jaacoppi** [comment]
In general, check whic executables you installed and if they're in your $PATH or not.

My first guess is you should be using espeak-ng, not espeak.

**Comment by ClockworkBastard** [comment]
After 5 minutes of tries, I've got "epeak-ng". And it does launches, but only a command line screen that can speak robot voice
